aws/table_aws_macie2_classification_job.go

```diff
@@ -177,6 +187,11 @@ func getMacie2ClassificationJob(ctx context.Context, d *plugin.QueryData, h *plu
 		id = d.KeyColumnQuals["job_id"].GetStringValue()
 	}
 
+	// empty check for job id
+	if id == "" {
```

Comment:
when would ID be empty?

Reply:
@cbruno10 user can provide empty value in get call - `select * from aws_macie2_classification_job where job_id = ''`. API should not be called with an empty value.